### PR TITLE
Card Styling updates for Mobile/ Desktop, addition of overview to cards

### PR DESCRIPTION
**Pull request by hzyniecki**
-  Refactor Reset password Modal
-  Added in movie overview to cards
-  Contain Home button in single css, instead of different hyperlinks
-  Fixed modal on the nomination window for mobile
-  Disabled add movie when you’re not logged
-  Cleaned up nomination creation toaster message
-  Fixed dropdown location for movie search in mobile view
-  Created media queries for add nomination button
-  Removed Reset Modal css file
-  Removed unused code

### CODE REVIEW

File: src/modals/NewNominationModal.tsx [hunk 1]
```diff
@@ -268,43 +291,17 @@ export default function NewNominationModal(props: NewNominationProps) {
...
-
-                        <List hidden={movieOptions.length === 0} sx={listStyle}>
+                        {/* Search Results */}
+                        <List hidden={movieOptions.length === 0} sx={searchListStyle}>
                             {
                                 movieOptions.map(option => (
                                     <>
                                         <ListItem alignItems='flex-start' onClick={() => {
                                             updateMovieSelection(option);
+                                            console.log(option, 'option');
```
Comment (CarterW715) [inline src/modals/NewNominationModal.tsx]: remove console.logs :)

File: src/modals/NewNominationModal.tsx [hunk 2]
```diff
@@ -45,17 +45,32 @@ interface nominationForm {
...
-    width: '100%',
     maxWidth: 360,
-    bgcolor: 'background.paper'
+    bgcolor: 'background.paper',
+    borderStyle: 'solid',
+    borderColor: '#eaeaea',
+    bottom: '-4rem',
+    top: '8rem',
+    // Desktop
+    '@media (max-width:960px)': {
```
Comment (CarterW715) [inline src/modals/NewNominationModal.tsx]: use material ui breakpoints

Comment (hzyniecki) [reply src/modals/NewNominationModal.tsx]: https://mui.com/material-ui/customization/breakpoints/

File: src/modals/NewNominationModal.tsx (hunk 1 above)
Comment (hzyniecki) [reply src/modals/NewNominationModal.tsx]: Should be gone now

File: src/modals/NewNominationModal.tsx (hunk 2 above)
Comment (hzyniecki) [reply src/modals/NewNominationModal.tsx]: Fixed

File: src/index.tsx [hunk 3]
```diff
@@ -18,6 +18,15 @@ const theme = createTheme({
         button: {
             fontFamily: "SoraBold"
         }
+    },
+    breakpoints: {
```
Comment (CarterW715) [inline src/index.tsx:22]: Did these need to be changed?

**Pull request closed without merging**